app/src/main/java/com/wizeline/compose/academy/ui/CustomButton.kt

```diff
@@ -0,0 +1,21 @@
+package com.wizeline.compose.academy.ui
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun CustomButton(modifier: Modifier, label: String) {
+    Button(onClick = { /*TODO*/ }, shape = MaterialTheme.shapes.medium, modifier = modifier) {
```

Comment:
Also here remember that we need to create our functions in the max top hight level we should be pass the onClick method like a parameter ☘️